R/object_usage_linter.R

```diff
@@ -268,30 +275,26 @@ parse_check_usage <- function(expression,
       function_name,
       capture(
         name = "message",
-        anything,
-        one_of(quote, "\u2018"),
-        capture(name = "name", anything),
-        one_of(quote, "\u2019"),
-        anything
+        zero_or_more(any, type = "lazy"),
+        maybe(
+          one_of(quote, "\u2018"),
```

Comment:
Since we use `withr::with_options()` to disable fancy quotes now, this could be simplified to `"'"`. WDYT?

Reply:
SGTM, let's do it